egs/hub4_spanish/s5/local/expand_graphemic_lexicon.sh

```diff
@@ -0,0 +1,64 @@
+#! /bin/bash
```

Comment:
I guess not. It's just there to properly generate a new lexicon from extra text when you have already trained the acoustic models, which was what Chunxi was doing. Should I get rid of it?